integration-tests/cdi-security-deny-all-jaxrs/pom.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
Does this really need its own integration test modules? Most of this testing should be done as unit testing using QuarkusUnitTest, and native image testing should be possible in one of the existing security modules that already use this functionality.

It is definitely not appropriate to have one integration test module per interceptor, that is way too fine grained an approach for integration tests.

Reply:
Config switches for denying all JAX-RS or denying unnanotated, they have to be different for the tests. Is there a way to have one test module that would test different configuration combinations?
Or do you mean that the switches should not be tested in integration tests?

Reply:
Have a look at QuarkusUnitTest. These tests are in the deployment modules, and each test can start Quarkus with a different config. There are some security related ones in the vertx-http module.

Reply:
Anyway, will do it tomorrow

Reply:
yea, so you should add a smoke test into one of the existing security integration tests. We can't add a new module for each discrete feature because otherwise our CI will take hours to run.

Reply:
done. One test added to the main integration tests, the rest is moved to deployment modules.